src/openapi/streaming/subscription.js

```diff
@@ -19,7 +19,7 @@ import SerializerFacade from './serializer-facade';
 /**
  * The static counter to generate unique reference id's.
  */
-let referenceIdCounter = 0;
+let referenceIdCounter = 1;

```

Comment:
should make no difference but this is just in case since the problems occur on subscription id 0

Reply:
@lukeapage It would be good to confirm with openapi that they don't rely on ReferenceId starting from 0.